tests/sorting_products.js

```diff
@@ -1,76 +1,45 @@
 import loginPage from "../Pages/loginPage.js";
 import productsPage from "../Pages/productsPage.js";
-const validUserName = 'standard_user';
-const validPassword = 'secret_sauce';
+import { validUser } from "../Pages/users.js";
 const baseUrl = 'https://www.saucedemo.com/';
 
 
 fixture`Verify sorting options`
     .page(baseUrl)
 
-  //MAKES SENSE: (A TO Z, Z to A, si preturi) sort simplu, click pe optin, luat lista, comparat liste.
     
-test(`Sort products by Name (A to Z)`, async t => { 
-    await loginPage.typeUserName(validUserName);
-    await loginPage.typePassword(validPassword);
-    await loginPage.clickOnLoginBttn();
-  
-    const productTitleElements = await productsPage.productTitle;
-    const productTitles = [];
-  
-    for (let i = 0; i < await productTitleElements.count; i++) {
-      const title = await productTitleElements.nth(i).innerText;
-      console.log(`Title ${i}: ${title}`);
-      productTitles.push(title);
-    }
-  
-    console.log('Product titles: ', productTitles);
+test(`Sort products by Name (A to Z)`, async (t) => {
+  await loginPage.signIn(validUser);
+
+  const initialTitles = await productsPage.getProductTitles(); 
+
+  await productsPage.openSortDrop();
+  await productsPage.sortByAscending();
+
+  const copyTitles = [...initialTitles];
```

Comment:
You need to sort the initialTitles and get the list of titles once again and compare those two lists.